examples/experimental/onnx/semantic_segmentation/onnx_ptq_segmentation.py

```diff
@@ -25,7 +25,7 @@
 
 def run(onnx_model_path: str, output_model_path: str, dataset_name: str,
         dataset_path: str, num_init_samples: int,
-        input_shape: List[int], ignored_scopes: List[str] = None):
+        input_shape: List[int], ignored_scopes: List[str] = []):
```

Comment:
```suggestion
        input_shape: List[int], ignored_scopes: Optional[List[str]] = None):
```
Please don't use mutable variable as a default value: https://docs.python-guide.org/writing/gotchas

It would be also helpful to fix all similar type hints issues if any in a single PR.

Reply:
Yes, good patch.